spec/features/families/admin_views_all_inactive_families_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+feature "Admin can see all the not active families when in families#index" do
+  context "when click on the 'Not Active' button" do
+    scenario "successfully", js: true do
```

Comment:
Since we made the views work without JS, we no longer need the `js: true` here

Reply:
Done in sha `f995a5f`